crates/dojo-core/src/world_test.cairo

```diff
@@ -354,47 +327,36 @@ fn test_execute_origin_failing() {
     let world = deploy_world();
 
     world.register_system(origin::TEST_CLASS_HASH.try_into().unwrap());
-    world.register_system(origin_wrapper::TEST_CLASS_HASH.try_into().unwrap());
     world.register_component(foo::TEST_CLASS_HASH.try_into().unwrap());
     let data = ArrayTrait::new();
 
     let eve = starknet::contract_address_const::<0x1338>();
-    world.execute('origin_wrapper', data);
+    world.execute('origin', data);
 }
 
 #[test]
 #[available_gas(6000000)]
 fn test_execute_multiple_worlds() {
-    // Deploy executor contract
-    let executor_constructor_calldata = array::ArrayTrait::new();
-    let (executor_address, _) = deploy_syscall(
-        executor::TEST_CLASS_HASH.try_into().unwrap(),
-        0,
-        executor_constructor_calldata.span(),
-        false
-    )
-        .unwrap();
-
     // Deploy world contract
     let mut constructor_calldata = array::ArrayTrait::new();
```

Comment:
let's remove the `world_factory` entirely

Reply:
Not sure I understand this one... world_factory files have been removed.

Reply:
ah my bad misread